GTG/gtk/editor/taskview.py

```diff
@@ -463,6 +463,37 @@ def insert_tag(self, tag):
         line_mark = self.buff.create_mark("lastline", lastline, False)
         self.insert_at_mark(self.buff, line_mark, tag)
 
+    def remove_tag_editor(self, tagname):
+        '''
+        TODO(jakubbrindza): Finish works on this function which will
+        enable removing a tag from a task in task editor.
+        '''
+        self.buff = self.get_buffer()
+        mark_start = self.buff.get_mark(tagname)
+        mark_end = self.buff.get_mark("/%s" + tagname)
+
+        it = self.buff.get_iter_at_mark(mark_start)
+        for tag in it.get_toggled_tags():
+            if hasattr('is_tag', tag) and tag.tagname == tagname:
+                        if mark_start:
+                            offset1 = self.buff.get_iter_at_mark(mark_start)\
+                                            .get_offset()
+                            if mark_start.get_offset() <= offset1 <= \
+                                    mark_end.get_offset():
+                                self.buff.delete_mark_by_name(tagname)
+                        if mark_end:
+                            offset2 = self.buff.get_iter_at_mark(mark_end)\
+                                            .get_offset()
+                            if mark_start.get_offset() <= offset2 <= \
+                                    mark_end.get_offset():
+                                self.buff.delete_mark_by_name("/%s" % tagname)
+
+        it_end = self.buff.get_iter_at_mark(mark_end)
+        self.buff.delete_text(it, it_end)
```

Comment:
This method doesn't exist. However, there is [delete](http://lazka.github.io/pgi-docs/Gtk-3.0/classes/TextBuffer.html#Gtk.TextBuffer.delete) method.